### ISSUE TITLE
基于PaddleDetection的YOLOv3_darknet模型训练报错RuntimeWarning: Failed to load model/variables

### ISSUE THREAD

**Issue by claudehotline** [PaddlePaddle/PaddleDetection#2462]
**PaddlePaddle 2.0.0
系统环境 window10， python 3.7.10**

**cmd窗口运行命令python tools/train.py -c configs/yolov3_darknet.yml报错：**

```
(paddlepaddle) G:\PaddleDetection>python tools/train.py -c configs/yolov3_darknet.yml
C:\anaconda3\envs\paddlepaddle\lib\site-packages\paddle\fluid\layers\utils.py:26: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  def convert_to_list(value, n, name, dtype=np.int):
C:\anaconda3\envs\paddlepaddle\lib\site-packages\paddle\fluid\framework.py:686: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  elif dtype == np.bool:
2021-03-29 22:14:03,343-INFO: If regularizer of a Parameter has been set by 'fluid.ParamAttr' or 'fluid.WeightNormParamAttr' already. The Regularization[L2Decay, regularization_coeff=0.000500] in Optimizer will not take effect, and it will only be applied to other Parameters!
W0329 22:14:04.025139 14172 device_context.cc:362] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 11.2, Runtime API Version: 10.2
W0329 22:14:04.057139 14172 device_context.cc:372] device: 0, cuDNN Version: 7.6.
C:\anaconda3\envs\paddlepaddle\lib\site-packages\paddle\fluid\io.py:2134: RuntimeWarning: Failed to load model/variables `['stage.0.0.0.bn.mean']`, please make sure model/variables file is saved with the following APIs: save_params, save_persistables, save_vars.
  warnings.warn(error_str % filenames, RuntimeWarning)
C:\anaconda3\envs\paddlepaddle\lib\site-packages\paddle\fluid\io.py:2134: RuntimeWarning: Failed to load model/variables `['stage.0.0.0.bn.offset']`, please make sure model/variables file is saved with the following APIs: save_params, save_persistables, save_vars.
  warnings.warn(error_str % filenames, RuntimeWarning)
C:\anaconda3\envs\paddlepaddle\lib\site-packages\paddle\fluid\io.py:2134: RuntimeWarning: Failed to load model/variables `['stage.0.0.0.bn.scale']`, please make sure model/variables file is saved with the following APIs: save_params, save_persistables, save_vars.
  warnings.warn(error_str % filenames, RuntimeWarning)
C:\anaconda3\envs\paddlepaddle\lib\site-packages\paddle\fluid\io.py:2134: RuntimeWarning: Failed to load model/variables `['stage.0.0.0.bn.var']`, please make sure model/variables file is saved with the following APIs: save_params, save_persistables, save_vars.
  warnings.warn(error_str % filenames, RuntimeWarning)
C:\anaconda3\envs\paddlepaddle\lib\site-packages\paddle\fluid\io.py:2134: RuntimeWarning: Failed to load model/variables `['stage.0.0.0.conv.weights']`, please make sure model/variables file is saved with the following APIs: save_params, save_persistables, save_vars.
  warnings.warn(error_str % filenames, RuntimeWarning)
C:\anaconda3\envs\paddlepaddle\lib\site-packages\paddle\fluid\io.py:2134: RuntimeWarning: Failed to load model/variables `['stage.0.0.1.bn.mean']`, please make sure model/variables file is saved with the following APIs: save_params, save_persistables, save_vars.
```

**yolov3_darknet.yml文件内容如下：**
architecture: YOLOv3
use_gpu: true
max_iters: 5000
log_iter: 20
save_dir: output
snapshot_iter: 200
metric: COCO
pretrain_weights: https://paddle-imagenet-models-name.bj.bcebos.com/DarkNet53_pretrained.tar
weights: output/yolov3_darknet/model_final
num_classes: 80
use_fine_grained_loss: false

YOLOv3:
  backbone: DarkNet
  yolo_head: YOLOv3Head

DarkNet:
  norm_type: sync_bn
  norm_decay: 0.
  depth: 53

YOLOv3Head:
  anchor_masks: [[6, 7, 8], [3, 4, 5], [0, 1, 2]]
  anchors:
    [
      [10, 13],
      [16, 30],
      [33, 23],
      [30, 61],
      [62, 45],
      [59, 119],
      [116, 90],
      [156, 198],
      [373, 326],
    ]
  norm_decay: 0.
  yolo_loss: YOLOv3Loss
  nms:
    background_label: -1
    keep_top_k: 100
    nms_threshold: 0.45
    nms_top_k: 1000
    normalized: false
    score_threshold: 0.01

YOLOv3Loss:
  ignore_thresh: 0.7
  label_smooth: true

LearningRate:
  base_lr: 0.001
  schedulers:
    - !PiecewiseDecay
      gamma: 0.1
      milestones:
        - 400000
        - 450000
    - !LinearWarmup
      start_factor: 0.
      steps: 4000

OptimizerBuilder:
  optimizer:
    momentum: 0.9
    type: Momentum
  regularizer:
    factor: 0.0005
    type: L2

_READER_: "yolov3_reader.yml"

**Comment by yghstill** [comment]
@claudehotline 看报错是接口不兼容问题，你是基于PaddleDetection哪个分支的代码训练的呢？请使用PaddleDetection的release 2.0-rc最新代码再重新训练试下呢？

**Comment by claudehotline** [comment]
找到原因了，是中文路径的问题
